inc/saivlan.h

```diff
@@ -474,6 +474,47 @@ typedef enum _sai_vlan_member_attr_t
      */
     SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE,
 
+    /**
+     * @brief Indicates if the bridge port is set to drop the Tunnel Terminated broadcast, unknown unicast and multicast traffic
+     *
+     * When set to true, egress BUM traffic will be dropped
+     * Valid only when the SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID is of type PORT.
+     * Valid only for .1Q bridge ports.
```

Comment:
Can the comment be made explicit and state it as below to include both port and lag ?
* Valid only when the SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID is of type SAI_BRIDGE_PORT_TYPE_PORT

Reply:
Ack